skyportal/handlers/api/allocation.py

```diff
@@ -366,6 +369,19 @@ def my_autopct(pct):
                 ).scalar()
                 values.append(total_matches)
 
+                followup_requests = session.scalars(stmt).all()
+                phase_angles = []
+                for followup_request in followup_requests:
+                    try:
+                        status_split = followup_request.status.split(" ")[-1]
+                        status_split = status_split.replace("_", ":").replace(" ", "T")
+                        tt = astropy.time.Time(status_split, format='isot')
+                        phase_angle = moon_phase_angle(tt)
+                        phase_angles.append(phase_angle.value)
+                    except Exception:
+                        pass
+                phases.append(phase_angles)
```

Comment:
Is this meant to be a nested list of lists or one long list?

Reply:
Yeah nested.